src/lib.rs

```diff
@@ -33,7 +33,8 @@
 //! let ref mut w = BufWriter::new(file);
 //!
 //! let mut encoder = png::Encoder::new(w, 2, 1); // Width is 2 pixels and height is 1.
-//! encoder.set(png::ColorType::RGBA).set(png::BitDepth::Eight);
+//! encoder.set_color(png::ColorType::RGBA);
+//! encoder.set_depth(png::BitDepth::Eight);
```

Comment:
This example is marked `ignore`, that's why CI hasn't caught it before. Replacing this with `no_run` might be better.